fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1 @@
+../../../../../.rubocop.yml
```

Comment:
Good point, this won't work when the gem is actually installed, only when using it via `:path` from the #monorepo. I wanted to avoid duplicating that file — any good ideas?